README.md

```diff
@@ -632,46 +632,178 @@ A GraphQL server can act as a Gateway that composes the schemas of the underlyin
 
 In Gateway mode the following options are not allowed (the plugin will throw an error if any of them are defined):
 
-* `schema`
-* `resolvers`
-* `loaders`
+- `schema`
+- `resolvers`
+- `loaders`
 
 Also, using the following decorator methods will throw:
 
-* `app.graphql.defineResolvers`
-* `app.graphql.defineLoaders`
-* `app.graphql.replaceSchema`
-* `app.graphql.extendSchema`
+- `app.graphql.defineResolvers`
+- `app.graphql.defineLoaders`
+- `app.graphql.extendSchema`
 
 ```js
 const gateway = Fastify()
+const GQL = require('fastify-gql')
 
 gateway.register(GQL, {
   gateway: {
-    services: [{
-      name: 'user',
-      url: 'http://localhost:4001/graphql',
-      rewriteHeaders: (headers) => {
-        if (headers.authorization) {
-          return {
-            authorization: headers.authorization
+    services: [
+      {
+        name: 'user',
+        url: 'http://localhost:4001/graphql',
+        rewriteHeaders: (headers) => {
+          if (headers.authorization) {
+            return {
+              authorization: headers.authorization
+            }
           }
-        }
 
-        return {
-          x-api-key: 'secret-api-key'
+          return {
+            'x-api-key': 'secret-api-key'
+          }
         }
+      },
+      {
+        name: 'post',
+        url: 'http://localhost:4002/graphql'
       }
-    }, {
-      name: 'post',
-      url: 'http://localhost:4002/graphql'
-    }]
+    ]
   }
 })
 
 await gateway.listen(4000)
 ```
 
+#### Automatically refresh schema inside Gateway mode
+
+Service which uses Gateway mode can obtain a new version of schema automatically. A developer should provide a few additional parameters into the Gateway config:
```

Comment:
I'd like to propose a change to description, something along the lines of:

`The Gateway service can obtain new versions of federated schemas automatically within a defined polling interval using the following configuration:`